### PR TITLE
[MLOBS] Update IO in API docs

### PR DESCRIPTION
**Pull request by lievan**
### What does this PR do? What is the motivation?

Add documentation for `Document` IO type and add clarification for setting IO values 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by Yun-Kim — APPROVED** [review]

File: content/en/llm_observability/setup/api.md [hunk 1]
```diff
@@ -141,15 +141,25 @@ If the request is successful, the API responds with a 202 network code and an em
 #### IO
 | Field   | Type   | Description  |
 |---------|--------|--------------|
-| value   | string | Input or output value. This should be used for all spans except for LLM spans. |
+| value   | string | Input or output value. If not set, this value will be inferred from messages or documents. |
```
Comment (aliciascott) [inline content/en/llm_observability/setup/api.md:144]:
```suggestion
| value   | string | Input or output value. If not set, this value is inferred from messages or documents. |
```

File: content/en/llm_observability/setup/api.md [hunk 2]
```diff
@@ -141,15 +141,25 @@ If the request is successful, the API responds with a 202 network code and an em
...
 | messages| [[Message](#message)] | List of messages. This should only be used for LLM spans. |
```
Comment (aliciascott) [inline content/en/llm_observability/setup/api.md:145]:
```suggestion
| messages| [Message](#message) | List of messages. This should only be used for LLM spans. |
```

File: content/en/llm_observability/setup/api.md [hunk 3]
```diff
@@ -141,15 +141,25 @@ If the request is successful, the API responds with a 202 network code and an em
...
+| documents| [[Document](#document)] | List of documents. This should only be used as the output for retrieval spans |
```
Comment (aliciascott) [inline content/en/llm_observability/setup/api.md:146]:
```suggestion
| documents| [Document](#document) | List of documents. This should only be used as the output for retrieval spans |
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @lievan some small changes on this one too, don't need the extra bracket for the links, let me know once updated and I'll approve and merge.

**Review by aliciascott — APPROVED** [review]
Hi @lievan just re-approved after the last changes, thanks! Is this ready to be merged ?

**Comment by lievan** [comment]
Yep this is ready to be merged, thanks! @aliciascott

**Pull request merged**